tests/tests/cpld_fv3_ccpp_384_mom6_cice_ww3_35d_2012010100

```diff
@@ -0,0 +1,83 @@
+#
+#  CPLD_FV3_CCPP_384_MOM6_CICE_WW3_35D_BMARK_RT test
+#
+
+export TEST_DESCR="Coupled FV3-CCPP-MOM6-CICE-WW3 system"
+
+export CNTL_DIR="RT-Baselines_35d_bmwav_2012"
+export CNTLMED_DIR="RT-Baselines_35d_bmwav_2012/RESTART"
+
+export LIST_FILES=""
```

Comment:
Does this line do anything?

Reply:
They are not used. I can remove them. Thanks.

Reply:
When I tried to run the warm 35d tests, I get `CNTL_DIR: unbound variable` error. Looks like these lines, even though they may not be used, need to be in place.

Reply:
When I was doing the cmeps commit, I had added some empty variables to default_vars (which I don't think I ended up using):

# default mediator component (NEMS mediator)
export MEDCOMP=''
export RESTART_PREFIX=''
export RESTART_SUFFIX=''
}

I think you could do the same thing here? Add CNTL_DIR as an empty string in default_vars.

Reply:
Yes. If that does not work, it will have to be fixed from the script side.